src/LobbyAPI.hs

```diff
@@ -13,9 +13,9 @@ import LobbyServer as Server
 -- |The api provided by the server.
 data LobbyAPI = LobbyAPI
   { connect :: Remote (String -> Server ())
-  , createGame :: Remote (Server (Maybe (String)))
+  , createGame :: Remote (Int -> Server (Maybe (String)))
```

Comment:
What's the Int for. I'm guessing max number of players in game.
Comment the function